### DISCUSSION TITLE
GIT_EEOF when accessing GitHub

### DISCUSSION THREAD

**Discussion by Uncommon** [libgit2/libgit2#6389] [Q&A]
Just wondering if anyone else has experience with this - when attempting to access a GitHub private repository via https with a personal access token, `git_remote_connect()` errors out with `GIT_EEOF`. This suggests that the server is sending an incomplete response, but libgit2 could be parsing it wrong.

I'm going to try to keep debugging it, but I thought I'd throw this out there in case someone else has run into it. Any thoughts?

**Comment by ethomson** [comment]
I haven't seen this. Is it any repository? Only one repository? Always the same one? etc.

**Reply by Uncommon** [reply]
So far just one particular repository that I'm testing with. I'll try some others and see how it goes.

**Reply by Uncommon** [reply]
I made a test repo on bitbucket.org and I'm getting the same error.

**Reply by ethomson** [reply]
Interesting. I don't suppose you could share that with me.

**Reply by Uncommon** [reply]
I emailed it to you.
